apps/dashboard/src/context/environment/environment-context.tsx

```diff
@@ -0,0 +1,14 @@
+import type { IEnvironment } from '@novu/shared';
```

Comment:
I'd keep context and provider in the same files to remove the need for another nesting level. That is `/context/environmentContext`

Reply:
Yes, I did the same initially but got the eslint error saying: `Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.`

That's why I moved it to a separate "context" file, but I'm still seeing the same error because of this `{ EnvironmentContext, useEnvironmentContext };` but we don't care much about fast refresh in the "context" file as the code will be constant.